package helpers

func GenerateError(err error) error {
	return err
}
